bridge/irc_connection.go

```diff
@@ -40,6 +56,9 @@ func (i *ircConnection) OnWelcome(e *irc.Event) {
 
 	i.JoinChannels()
 
+	// just in case NickServ, Q:Lines, or otherwise force our nick to be not what we expect!
+	i.manager.puppetNicks[i.GetNick()] = i
```

Comment:
This is going to be a lot of round-trip calls when the bot first starts. I think this should be fixed (in a later PR), once we have networked varys somewhat working.